sklearn/notebooks/census_training/train.py

```diff
@@ -0,0 +1,166 @@
+# %load iris.py
+import datetime
+import os
+import pandas as pd
+import subprocess
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.externals import joblib
+from sklearn.feature_selection import SelectKBest
+from sklearn.pipeline import FeatureUnion
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelBinarizer
+
+
+# TODO: REPLACE 'true-ability-192918' with your GCS BUCKET_ID
+BUCKET_ID = 'true-ability-192918'
+
+
+# ---------------------------------------
+# 1. Add code to download the data from GCS (in this case, using the publicly hosted data).
+# ML Engine will then be able to use the data when training your model.
+# ---------------------------------------
+census_data_filename = 'adult.data'
+census_test_filename = 'adult.test'
+data_dir = 'gs://cloud-samples-data/ml-engine/sklearn/census_data'
+subprocess.check_call(['gsutil', 'cp',
+                       os.path.join(data_dir,census_data_filename),
+                       census_data_filename])
+subprocess.check_call(['gsutil', 'cp',
+                       os.path.join(data_dir, census_test_filename),
+                       census_test_filename])
+
+
+# ---------------------------------------
+# This is where your model code would go.
```

Comment:
Is it plausible to put an example of "model code" in comments here? Or at least more of an explanation of exactly what you mean by "model code".

Reply:
The model code is everything below. 

I'll switch to: "This is where your model code would go. Below is an example model using the census dataset.